cherryontech/jupiter-jumpers
# Addresses various issues

## Description

1. Addresses various issues

## Related Issue

#105 
#104 
#80 
#77 
#78 
#84 
#97 


## Testing Instructions

1. Create a new account:
2. In the summary view (for a single log), ensure the grey cloud (the graphic accompanying "There are no reflections") is not too large (desktop size)
3. In the summary view (for a single log), ensure the grey could is not too large (mobile size)
4. Open the Trends page, ensure there is a message inviting the user to start logging (since there are no mood logs yet)
5. With another account that has mood logs (or add some mood logs to this new account), check the trends page, and resize it to a mobile size and ensure the graphs looks okay (nothing is squished, etc.)
6. Open the 'date' filter in trends and ensure the font is Open Sans for all the components 
7. Create a new log, and click open the "Reflections" box, but don't write anything in the box. Save the log. Open its daily summary view, and ensure nothing shows up for the reflections section (the bug was that an empty log was being displayed if the user opened the box before)
